### ISSUE TITLE
Web chat content is way, way too narrow

### ISSUE THREAD

**Issue by denravonska** [zhukov/webogram#1411]
The actual content of the web chat occupies around 25% of the screen estate. Is there a concrete reason for why we want 75% whitespace and 25% content? Compare that to Slack which promotes content before.. I don't know, background?

![screenshot_2017-05-10_20-20-40](https://cloud.githubusercontent.com/assets/135439/25914846/e238e7b0-35bf-11e7-9335-340c604d2384.png)

![screenshot_2017-05-10_20-34-11](https://cloud.githubusercontent.com/assets/135439/25915033/6c8186b6-35c0-11e7-9c4f-e0a2e18d2978.png)

**Comment by zhukov** [comment]
@denravonska I see the lines are also wrapped in slack so the actual text content width is about the same.
It's uncomfortable to read very long lines of text.

**Comment by denravonska** [comment]
@zhukov nah, that's just the text on the cryptocointalk page that's wrapped. Other links and user text occupies the full width. The comfort is a personal preference, I guess.

It can probably easily be changed with a Greasemonkey script though.

**Comment by azzimuth** [comment]
+1
I use Slack at work, long text in never wrapped.

I would also like to see Webogram to use more space.

**Comment by Ryuno-Ki** [comment]
I agree with @zhukov. From a design point of view, the recommendation is to not allow the text to become too wide. I can look up the numbers if needed. It's a typography thing.

Web chat is meant as Direct Message here?

**Comment by guilhermecaldas** [comment]
Here are 4 examples of Telegram using the space of screen:
(2 for Telegram Desktop and 2 for Telegram for MacOS)
<img width="1280" alt="screen shot 2017-06-21 at 01 47 38" src="https://user-images.githubusercontent.com/3738460/27367796-4283f59e-5625-11e7-8d17-0f3c28cb8ed0.png">
---
---
<img width="1280" alt="screen shot 2017-06-21 at 01 48 08" src="https://user-images.githubusercontent.com/3738460/27367797-42871aee-5625-11e7-8d2b-238a6fb8a395.png">
---
---
<img width="1280" alt="screen shot 2017-06-20 at 22 55 36" src="https://user-images.githubusercontent.com/3738460/27367826-667c737c-5625-11e7-9ee9-3b97432ffaee.png">
---
---
<img width="1280" alt="screen shot 2017-06-20 at 22 55 48" src="https://user-images.githubusercontent.com/3738460/27367827-667c4bcc-5625-11e7-9991-d6cfd519a8d1.png">

The only rule both use is to fill as much space as possible, for the windows. Text blocks can be wrapped or fill all width. Would not be better if text wrap was available in settings for disable or enable?

**Comment by moeinomidii** [comment]
@zhukov ] thing design of telegram app in mac & win is so much better than the web version there is a lot of issues in it . Emojis have not been updated ... i cant see f**k in web version  :-1:

**Comment by stek29** [comment]
@moeinomidii thanks for letting us know how much fuck you see in Webogram.

**Comment by moeinomidii** [comment]
@stek29 Sometimes I receive and send more :-)

**Comment by jvschiavo** [comment]
Could we just update the design of TGWeb to match Desktop and mobile versions please?

**Comment by rodneyrod** [comment]
There could be a toggle setting to allow people to change view from the padded version currently in place, or to a 'widescreen' version that fills out the sidebars of whatever monitor it's displayed on.

**Comment by axelchalon** [comment]
If you have a web browser extension for handling user stylesheets, you can simply use the following CSS to make Webogram full-screen:
```css
.im_page_wrap { max-width: none; height: 100vh; }
.im_page_wrap > * { height: 100%; }
.im_message_wrap { max-width: none; }
.tg_head_split { max-width: none; }
.footer_empty { height: 0; }
```

**Comment by rodneyrod** [comment]
@last-partizan Cheers for adding that, but would it be possible to extend it all the way to the edges with no padding, and not affecting image / sticker sizes?

**Comment by last-partizan** [comment]
@rodneyrod tecnically - everything is possible.

But i don't think it's a good idea.

(Anyway, looking by the state of the project i ended up using personal fork of webogram, and you can do it to with any fixes you want)

**Comment by rodneyrod** [comment]
@last-partizan How would that change be done exactly? I've tried to push CSS changes to extend the boarder and remove the padding, however as a side effect it also reduced the size of all images to a half or even a quarter of their original size in regards to stickers.

Is this a known issue with Webogram that would prevent pushing this change?
